tests/fixtures/saas/gladly_fixtures.py

```diff
@@ -0,0 +1,67 @@
+import time
+from typing import Any, Dict
+
+import pydash
+import pytest
+import requests
+from requests.auth import HTTPBasicAuth
+
+from tests.ops.integration_tests.saas.connector_runner import (
+    ConnectorRunner,
+    generate_random_email,
+    generate_random_phone_number,
+)
+from tests.ops.test_helpers.vault_client import get_secrets
+
+secrets = get_secrets("gladly")
+
+
+@pytest.fixture(scope="session")
+def gladly_secrets(saas_config) -> Dict[str, Any]:
+    return {
+        "domain": pydash.get(saas_config, "gladly.domain") or secrets["domain"],
+        "account_email": pydash.get(saas_config, "gladly.account_email")
+        or secrets["account_email"],
+        "api_key": pydash.get(saas_config, "gladly.api_key") or secrets["api_key"],
+    }
+
+
+@pytest.fixture(scope="session")
+def gladly_identity_email(saas_config) -> str:
+    return pydash.get(saas_config, "gladly.identity_email") or secrets["identity_email"]
+
+
+##Should we use a pre-determined identity email or generate a random one?
```

Comment:
We prefer a random one, as long as we can create data for it as part of the fixtures